Changelog.md

```diff
@@ -2,6 +2,7 @@
 
  * Add `takeWhileEnd`, `dropWhileEnd` and `strip`  for strict bytestrings
  * Add `IsList` instances
+ * Deprecate `Data.ByteString.Lazy.Builder`
```

Comment:
AFAIK deprecations usually cause a major version bump. Is there a good reason to make an exception in this case?

Reply:
The current PVP on pvp.haskell.org requires a major version bump, but it turns out to be highly undesiable (see https://github.com/haskell/network/issues/401 and https://github.com/haskell/pvp/issues/12). The upstream version 1.1 now says that the major versions may remain the same: https://github.com/haskell/pvp/blob/master/v1.1/pvp-specification.md

Reply:
Thanks! I think I currently tend to "play it safe" and include this deprecation in the major version bump (which I expect to happen within the next year).

But I don't really mind too much.

@Bodigrim, @hsyl20 What do you think?

Reply:
As discussed in haskell/pvp#12 the original motivation for considering deprecations as a breaking change is not valid anymore. I agree with @fumieval here that it's acceptable (and preferable) to treat it as a minor bump.

Reply:
FWIW, there are zero(!?) occurrences of Data.ByteString.Lazy.Builder on GitHub, so it should be safe anyway.

Reply:
I'm convinced. Thanks! :)